Neos.Flow/Tests/Behavior/Features/Bootstrap/SecurityOperationsTrait.php

```diff
@@ -206,6 +206,8 @@ protected function setupSecurity()
         $this->accountRepository = $this->objectManager->get(Security\AccountRepository::class);
 
         $this->authenticationManager = $this->objectManager->get(AuthenticationProviderManager::class);
+        // Making sure providers and tokens were actually build, so the singleton TestingProvider exists.
+        $this->authenticationManager->getProviders();
```

Comment:
Mh.. Requiring the call of a getter is a bit smelly, no?

Reply:
No because actually the provider shouldn't be a singleton. It's just the tests that do nasty things, because they manipulate the internal state of the (test) provider (which is not possible if it was not build at that time).

Reply:
👍 I probably missed out that this was for tests only the first time around